pkg/network/ebpf/c/protocols/kafka/kafka-classification.h

```diff
@@ -165,9 +164,37 @@ static __always_inline bool skip_varint_number_of_topics(pktbuf_t pkt, u32 *offs
     return true;
 }
 
+static __always_inline __maybe_unused bool skip_varint(pktbuf_t pkt, u32 *offset, u32 max_bytes) {
```

Comment:
nit: worth adding documentation to the function

Reply:
Done.